exporter/otlpexporter/README.md

```diff
@@ -28,6 +28,16 @@ The following settings can be optionally configured:
   [grpc.WithInsecure()](https://godoc.org/google.golang.org/grpc#WithInsecure).
 - `balancer_name`(default = pick_first): Sets the balancer in grpclb_policy to discover the servers.
 See [grpc loadbalancing example](https://github.com/grpc/grpc-go/blob/master/examples/features/load_balancing/README.md).
+- `timeout` (default = 5s): Is the timeout for each operation.
+- `retry_settings`
+  - `disabled` (default = false)
+  - `initial_backoff` (default = 5s): Time to wait after the first failure before retrying; ignored if retry_on_failure is false
```

Comment:
What is `retry_on_failure`? I think it could be the name of `retry_settings`?

Reply:
done.